src/app/space/settings/areas/areas.component.html

```diff
@@ -38,8 +38,12 @@ <h3 class="areas-header">Areas that make up the {{context.space.attributes.name
   </div>
 </div>
 
-<modal #createArea modalClass="chromeless-modal">
-  <modal-content class="chromeless-modal-content">
-    <create-area-dialog [host]="createArea" [parentId]="selectedAreaId" [areas]="areas" (onAdded)="addArea($event)"></create-area-dialog>
-  </modal-content>
-</modal>
+<div class="modal fade" bsModal #modal="bs-modal" tabindex="-1" role="dialog" aria-labelledby="mySmallModalLabel" aria-hidden="true" (onShown)="onShowHandler()">
```

Comment:
"mySmallModalLabel"? Where does this come from?

Reply:
Sorry, I forgot to drop that when cleaning things up. It's gone now.